examples/pytorch/rgcn/README.md

```diff
@@ -51,3 +43,14 @@ FB15k-237 in Filtered-MRR
 ```
 python link.py --gpu 0 --eval-protocol filtered
 ```
+
+Summary 
+-------
+### Entity Classification
+
+| Dataset       | Result 
+| ------------- | -------
+| aifb        | ~86.1    
+| mutag       | ~69.7   
+| bgs         | ~91.4    
+| am          | ~78.5
```

Comment:
Do you want to include these results? I think previously this example did not list the results?

Reply:
Yes, I can include all results in `README`